Site/SiteJwPlayerMediaDisplay.php

```diff
@@ -323,7 +329,7 @@ protected function getJavascript()
 			$javascript.= sprintf("\t%s.addSource(%s, %d, %s);\n",
 				$this->getJavascriptVariableName(),
 				SwatString::quoteJavaScriptString($source['uri']),
-				$source['width'],
+				($source['width'] == '') ? "''" : $source['width'],
```

Comment:
Because we are using `%d` `0` is getting output instead of `''`. We should either make the default `0` or not use `%d`.